login-fire-form.html

```diff
@@ -308,6 +309,21 @@
     },
 
     /**
+     * Cleans inputs (email + password)
+     */
+    _cleanInputs: function() {
+      this.$$("#emailInput").value = '';
+      this.$$("#passwordInput").value = '';
```

Comment:
To avoid duplicated code, this line could be replaced by `this._cleanPasswordInput();`.